config/generic.yml

```diff
@@ -18,13 +18,13 @@ module: core
 config:
   tickrate: 10
   import:
-    - ravestate_conio
```

Comment:
Please don't change the generic.yml. Instead, create a new luigi.yml that has only your whitelisted modules.